monai/bundle/workflows.py

```diff
@@ -207,10 +210,21 @@ def __init__(
         **override: Any,
     ) -> None:
         super().__init__(workflow=workflow)
+        if config_file is not None:
+            _config_path = Path(ensure_tuple(config_file)[0])
+            if _config_path.is_file():
+                config_file = config_file
```

Comment:
This appears not to do anything. Also we're not considering the case when `config_file` is a sequence of file paths since onyl the first is taken here?

Reply:
Thanks for pointing out, there's a mistake. I deleted the line 216. 
And `_config_path` is only used to get the `config_root_path`. A sequence of `config_file` is still under consideration.
https://github.com/Project-MONAI/MONAI/blob/6938718b757e0366b8fc6344a1e7dbafcf588bbd/monai/bundle/workflows.py#L235